extensions/map-layers-formats/src/Tools/MapFeatureInfoTool.ts

```diff
@@ -112,7 +117,7 @@ export class MapFeatureInfoTool extends PrimitiveTool {
   public readonly onInfoCleared =  new BeEvent();
 
   public static override toolId = "MapFeatureInfoTool";
-  public static override iconSpec = "icon-map";
+  public static override iconSpec = "data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxNiAxNiI+PHBhdGggZD0iTTUuODYyIDUuMTYyYTEuNDg4IDEuNDg4IDAgMSAxLTIuOTc1IDAgMS40ODggMS40ODggMCAwIDEgMi45NzUgMFpNMTIgN2MuNzEyIDAgMS4zODcuMTU0IDIgLjQyMlYwaC0xLjA1bC0yLjI3OCA3LjIwMUE0Ljk1IDQuOTUgMCAwIDEgMTIgN1ptLTQuNzkyIDYuMzQ5TDcgMTRIMFYwaDExLjQ2M0w4Ljg3MiA4LjEyOUM3LjczOSA5LjA0NSA3IDEwLjQyOSA3IDEyYzAgLjQ3LjA4Ni45MTcuMjA4IDEuMzQ5Wm0tLjAzMy04LjE4N1Y1LjA5YTIuNzY0IDIuNzY0IDAgMCAwLTIuODcyLTIuNzI4IDIuNzY0IDIuNzY0IDAgMCAwLTIuNzI4IDIuOEE2LjkwMyA2LjkwMyAwIDAgMCAyLjggOC4zMTJjLjQ3NC44NzcgMSAxLjcyNCAxLjU3NSAyLjUzOCAwIDAgLjk2My0xLjQ4OCAxLjU3NS0yLjUzN2E2LjkwMyA2LjkwMyAwIDAgMCAxLjIyNS0zLjE1Wk0xNiAxMmE0IDQgMCAxIDEtOCAwIDQgNCAwIDAgMSA4IDBabS00LjA1LTEuODVhLjQ2OS40NjkgMCAwIDAgLjUuNS42NTUuNjU1IDAgMCAwIC42NS0uNmwtLjAwMi0uMDMyQS40Ni40NiAwIDAgMCAxMi42IDkuNmEuNjE1LjYxNSAwIDAgMC0uNjUuNTVabTEuMTUgMy40NS0uMTUtLjJhMSAxIDAgMCAxLS41NS4zYy0uMSAwLS4xLS4wNS0uMDUtLjNsLjM1LTEuM2MuMTUtLjUuMS0uOC0uMTUtLjhhMy4yNSAzLjI1IDAgMCAwLTEuNjUuOGwuMS4yNWMuMTk4LS4xMjUuNDItLjIxLjY1LS4yNS4wNSAwIC4wNS4wNSAwIC4yNWwtLjMgMS4yYy0uMi43IDAgLjg1LjI1Ljg1YTIuODg1IDIuODg1IDAgMCAwIDEuNS0uOFoiLz48L3N2Zz4=";
```

Comment:
Why change this to a data URL? I can't find anywhere else in the code that uses one for its icon spec

Reply:
Mainly because I didn't want to through the process of adding a new icon to `@bentley/icons-generic-webfont`.

This was suggested by @ben-polinsky, I'm assuming this is a fine approach...

Reply:
So... not thrilled about the inlining of the data uri... while I pointed out it is possible, I'd much prefer something cleaner. The issue is that we (I) haven't thought through what a replacement would be and where it would go.

The icon doesn't have to be added to `@bentley/icons-generic-webfont`. More generally, the iconSpec string needs to be a class that some icon library recognizes and is included in the end application. This isn't great either, but we make the assumption that the icons-generic package is going to be in any front end app. 

Curious - where are you getting the icon from?

Reply:
UX team provided a SVG icon some time ago, that I converted to base64 encoding.

Reply:
You can use `icon-map`